extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/ColorSplasher.pushbutton/script.py

```diff
@@ -654,11 +640,12 @@ def __init__(self, categories, ext_ev, uns_ev, s_view, reset_event, ev_legend, e
         self._tableData.Columns.Add("Value", System.Object)
         names = [x._name for x in self.categs]
         self._tableData.Rows.Add("Select a Category Here!", 0 )
-        [self._tableData.Rows.Add(key_, value_ ) for key_, value_ in zip(names, self.categs)]
+        [self._tableData.Rows.Add(key_, value_) for key_, value_ in zip(names, self.categs)]
         self.out = []
         self.InitializeComponent()
     
     def InitializeComponent(self):
+        self._spr_top = System.Windows.Forms.Label()
```

Comment:
this is valid for almost each line below this: if you change the import as suggested at the beginning, there you can just use `Forms.XXX` and `Drawing.XXX`